README.md

```diff
@@ -20,6 +20,10 @@ y: |`md
 `|
 ```
 
+## Installation
+
+The VSCode extension for D2 can be installed using the extension sidebar or by going to the [Visual Studio Marketplace Website](https://marketplace.visualstudio.com/items?itemName=terrastruct.d2) and clicking 'Install'
```

Comment:
can you change to "Install", and add a period at the end? (after the quotes).

It's also overly verbose in the beginning. Just `You can install using...`